apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/app/GetNextBlocksTask.java

```diff
@@ -111,6 +111,11 @@ public GetNextBlocksTask(List<Long> blockIds, int start, int stop, int startHeig
             responseTime = System.currentTimeMillis() - startTime;
         }
 
+        if (response == null) {
+            log.debug("NULL GetNextBlocks response from peer: {}", peer.getAnnouncedAddress());
+            return null;
```

Comment:
May be better and safer to return :  List<BlockImpl> = Collections.emptyList() ?